source/NVDAObjects/IAccessible/ia2TextMozilla.py

```diff
@@ -684,9 +684,16 @@ def _findNextContent(self, origin, moveBack=False, limitToInline=False):
 				pass
 		return ti, obj
 
-	def move(self, unit, direction, endPoint=None):
+	def move(
+		self,
+		unit: int,
+		direction: str,
+		endPoint: str | None = None,
+	) -> int:
 		if direction == 0:
 			return 0
+		if self._makeRawTextInfo(self.obj, textInfos.POSITION_ALL)._getStoryLength() == 0:
```

Comment:
I think this would be less costly. We are sure that self.obj wil have an IAccessible2TextObject, right?

```suggestion
		if self.obj.IAccessibleTextObject.nCharacters == 0:
```